src/pages/api/github-load-notes.ts

```diff
@@ -64,7 +64,9 @@ export async function loadNotesFromUrl(url: string) {
         }
       }, {})
       note.type = 'root'
-      const contents = toMarkdown(note)
+      const contents = note.children
+        .map((node: any) => toMarkdown(node))
+        .join(' ')
```

Comment:
this was the culprit - we were only passing first node.

Reply:
good catch!!! 🌮